nixos/modules/services/hardware/keyd.nix

```diff
@@ -140,10 +188,8 @@ in
           config.users.groups.uinput.name
         ];
 
-        RuntimeDirectory = "keyd";
-
         # Hardening
-        CapabilityBoundingSet = [ "CAP_SYS_NICE" ];
+        # CapabilityBoundingSet = [ "CAP_SYS_NICE" ];
```

Comment:
This is not a complete list of capabilities, so just remove one if you don't need it :)